TGC/decisions/Colonisation-Asia.txt

```diff
@@ -312,12 +312,12 @@ political_decisions = {
 	organize_lebanon = {
 		picture = map_arabia
 		potential = {
+			civilized = yes
+			NOT = { has_global_flag = lebanon_organized }
```

Comment:
Any reason to put this back?

Reply:
Nope, just a difference from copy/pasting from a diff change 1-2 days ago on main rep